BUILDING.md

```diff
@@ -0,0 +1,158 @@
+# Building initrd for Ubuntu Core
```

Comment:
This is ambigious.

First half of the document describes how to rebuild core-initrd itself. When most developers do not need to do this. They can instead simply install `ubuntu-core-initramfs` prebuilt deb package from the snappy-dev/image ppa.

Second half of the document talks about creating new intird, but not including it into kernel snap. which is more of how to use.

People who need to do first or second things are not usually the same person. Thus it would be nice to have a doc that starts with simply installing ubuntu-core-initramfs deb and using it. And separate one, to rebuild ubuntu-core-initramfs from scratch if it needs any changes.

Most people do not need to rebuild ubuntu-core-initramfs deb at all, they simply install it prebuilt.